src/main/java/uk/gov/justice/digital/delius/jpa/standard/repository/MainOffenceRepository.java

```diff
@@ -7,6 +7,4 @@
 
 public interface MainOffenceRepository extends JpaRepository<MainOffence, Long> {
     List<MainOffence> findByOffenderId(Long offenderId);
-
```

Comment:
Is this safe? What about the offender main offence for a prior conviction?

Reply:
Yes, since a main offence can only exist in the  context of an event and event only exits when they have been convicted (though they can be created before they are sentenced). So finding by event Id really means `give me the offence associated with this conviction`

Reply:
But is this not 'find all my main offences including those related to the current event'?